src/components/Grid/styles/_20-mixins.scss

```diff
@@ -280,6 +271,15 @@
       }
     }
   }
+  $autoflow: map-get($data, autoflow);
```

Comment:
these mixins will still break as the variable declarations are being moved to bottom of the mixin block

Reply:
ah - the linter isn't actually catching them anymore, but this file is a hangover from prev commits. fixed

Reply:
tnx